docs/further.md

```diff
@@ -157,17 +157,14 @@ Some of these are [central snakemake command-line arguments determining its beha
   To reduce the burden of regular status checks,  this time automatically increases in 10 second steps with every status check whenever no finished jobs are found.
   But this will never go beyond a wait time of 180 seconds (3 minutes), to avoid long wait times once jobs do finish.
   Also, as soon as finished jobs are found, this gets reset to 40 seconds.
-  (TODO: respect the `--seconds-between-status-checks` option in the plugin and use it as the minimum wait time.)
 * With `--max-status-checks-per-second`, you can limit the frequency of individual attempts of querying the job status database within a round of status checks.
   Within a round, repeated attempts will happen whenever the `sacct` command used for the status query comes back with an error.
-  (TODO: set a reasonable default here; also double-check this is what we want to happen, here.)
```

Comment:
As with the TODO above, should we track this in a new issue? Maybe both can go into one issue to thoroughly review the wait times / frequencies settings in the plugin.

Reply:
OK, this is now in the discussion below, so I'll resolve here.